pkg/controller/master/node/node_down_controller.go

```diff
@@ -112,11 +127,21 @@ func (h *nodeDownHandler) OnNodeChanged(key string, node *corev1.Node) (*corev1.
 			return node, err
 		}
 
+		names, err := h.getPVCsName(&pod)
+		if err != nil {
+			return node, err
+		}
+		pvcSet.Append(names...)
+	}
+
+	if err := h.deleteVolumeAttachments(node.Name, pvcSet); err != nil {
+		return node, err
 	}
-	return node, nil
+
+	return h.resetHeartbeat(node)
 }
 
-func (h *nodeDownHandler) OnVMForceResetPolicyChanged(key string, setting *harvesterv1.Setting) (*harvesterv1.Setting, error) {
+func (h *nodeDownHandler) OnVMForceResetPolicyChanged(_ string, setting *harvesterv1.Setting) (*harvesterv1.Setting, error) {
```

Comment:
please double check previous

```
// if we haven't waited for vmForceResetPolicy.Period seconds, we enqueue event again
```

When policy is changed, the alreay in queue NODE will not be executed quickly/slowly.

Should we adjust the code, e.g.  enqueue after only 1 minute, thus the NODE can adapt to the new param ?

Reply:
I have re-tested the above logic(quickly/slowly) because `OnVMForceResetPolicyChanged` will re-enqueue if the period changes and replace the existing node obj.